src/Sulu/Bundle/ContactBundle/Contact/AccountManager.php

```diff
@@ -249,7 +249,7 @@ public function findContactsByAccountId($id, $locale, $onlyFetchMainAccounts = f
         if (!empty($contactsEntities)) {
             $contacts = [];
             foreach ($contactsEntities as $contact) {
-                $contacts[] = new Contact($contact, $locale, $this->tagManager);
+                $contacts[] = new Contact($contact, $locale);
```

Comment:
Contact Api Entity does not have the tagManager in its constructor: https://github.com/sulu/sulu/blob/1.6.7/src/Sulu/Bundle/ContactBundle/Api/Contact.php#L56